jobs/web/spec

```diff
@@ -852,11 +852,17 @@ properties:
       Dialing timeout, in Go duration format (1m = 1 minute). 0 means wait indefinitely.
     default: 5m
 
-  max_conns:
-    env: CONCOURSE_MAX_CONNS
+  api_max_conns:
+    env: CONCOURSE_API_MAX_CONNS
     description: |
-      The maximum number of open connections for a database connection pool.
-    default: 32
+      The maximum number of open connections for the api connection pool.
+    default: 10
```

Comment:
as mentioned in https://github.com/concourse/concourse-bosh-release/pull/66#pullrequestreview-312315186, if we go without a default here, it'd mean that we could go without having to worry about defaults changing in the binary, and then having to change them at the packaging level too 😁

Reply:
Good point! I'll change that